miden/README.md

```diff
@@ -73,11 +73,11 @@ To execute a program on Miden VM and generate a proof that the program was execu
 * `program: &Program` - a reference to a Miden program to be executed.
 * `stack_inputs: StackInputs` - a set of public inputs with which to execute the program.
 * `advice_provider: AdviceProvider` - an instance of an advice provider that yields secret, non-deterministic inputs to the prover.
-* `options: ProofOptions` - config parameters for proof generation. The default options target blake3-192 with 96-bit security level.
```

Comment:
Being RPO an option, shouldn't we specify that we are using blake? It is true the only `96` bits option is blake, but that might be a bit redundant for documentation as we also target people who are just introducing themselves to the stack.

Reply:
Hash function is just one of the options and I'm not sure it makes sense to call it out vs. some other options we've selected. In the future, we'll probably need to explain the values we selected for pre-sets in much more detail.